### ISSUE TITLE
Config: Validate that all command line options specific to scylla (not seastar) are included in scylla.yaml

### ISSUE THREAD

**Issue by slivne** [scylladb/scylladb#349]
- Validate that all command line options specific to scylla (not seastar) are
  included in scylla.yaml and are documented
- scylla.yaml is split into "Supported", "Not yet supported" - we are missing many scylla specific params (not seastar params) from the scylla.yaml (to mame a few --api-_, --enable-_, etc.), need to add them with a heading "Scylla additional parameters" (or something else that tzach provides)

**Comment by slivne** [comment]
@elcallio so did you add the scylla specific params to scylla yaml

**Comment by elcallio** [comment]
No, sorry, I got sidetracked and also wanted feedback on whether the above (not marked explicit no-go) stuff should really go in there.

**Comment by nyh** [comment]
I think everything we wanted to add to scylla.yaml, we added a long time ago, and there is nothing left to do here and this issue can be closed. Anybody thinks otherwise?

**Comment by denesb** [comment]
We now generate both the yaml parsing code and the command-line options from a single definition, so we can close this.
